### ISSUE TITLE
FileUploadEditor - get and stores original file name in database

### ISSUE THREAD

**Issue by rolembergfilho** [volkanceylan/Serenity#1005]
Hey guys!

In FileUploadEditor, how to get and store original file name in my database.

Example:

PrimaryImage stores the path of the image (done in MovieForm Tutorial)
PrimaryImageName stores the original name of the uploaded image (e.g. Keanu Reeves.png)

ps.: i found this attribute **OriginalNameProperty** in FileUploadEditor. What is it used for and how to use it?

**Comment by volkanceylan** [comment]
Set OriginalNameProperty to name of property / field that will store original name of file. That's all.

**Comment by rolembergfilho** [comment]
Allllmost close !
It's worked, but the original name of file just appeared after I saved the Entity.

Is there a way to show the original name of file before save?
E.g.: I selected a file. The original name of file is showed in this moment in his properly field.

Why? My secretary, while give attention to my patience, usually doesn't have a lot of attention to the form. So, when they picked the HealthCare authorization, she must wanna confirm if this file is correct or not.

It's important to show the original name of the file when they choose a file, before save the entity.
